interfaces/IBF-dashboard/src/assets/i18n/en.json

```diff
@@ -117,12 +113,8 @@
       "no-event-no-trigger": {
         "welcome": "Hello {{ name }}. To your right is the map of your country, where you can turn layers on and off. There are currently <strong>no alerts</strong> for dengue activated."
       },
-      "new-event-active-trigger": {
-        "welcome": "Hello {{ name }}. A {{ disasterTypeLabel }} alert for has been activated for <strong>{{ monthYear }}</strong>.",
-        "new-event-early-trigger": "The system however detects an alert as soon as <strong>{{ leadTime }} month(s) from now</strong>, which was not detectable earlier."
-      },
       "active-event-active-trigger": {
-        "welcome": "Hello {{ name }}. A {{ disasterTypeLabel }} alert has earlier been activated on <strong>{{ startDate }}</strong>. The trigger is activated for <strong>{{ monthYear }}</strong>."
+        "welcome": "Hello {{ name }}. A trigger warning for <strong>{{ disasterTypeLabel }}</strong> was activated on <strong>{{ startDate }}</strong>. The trigger is activated for <strong>{{ monthYear }}</strong>."
```

Comment:
@SwatiTh I also copied the new text structure to the separate 'dengue' and 'malaria' sections (which stupidly enough have separate sections here in en.json, while other disaster-types don't). This is unfortunately badly coded back in the day and should be improved in the future hopefully.